README.md

```diff
@@ -1,5 +1,9 @@
 # Pydata London Meetup Ballot
 
+[![Build Status](https://travis-ci.org/chfw/meetup_ballot.svg?branch=master)](https://travis-ci.org/chfw/meetup_ballot) [![Coverage](https://codecov.io/gh/chfw/meetup_ballot/branch/master/graph/badge.svg)](https://codecov.io/gh/chfw/meetup_ballot/branch/master)
```

Comment:
pydatalondon needs to enable it with travis and codecov, then replace chfw with PyDataLondon